WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -92,235 +100,134 @@
         </DataTemplate>
     </Page.Resources>
 
-    <StackPanel Spacing="10" HorizontalAlignment="Stretch">
+    <Grid HorizontalAlignment="Stretch">
```

Comment:
The spacing at the bottom looks quite odd now:

![Screenshot of IconsPage](https://user-images.githubusercontent.com/16122379/217064543-cd1666e1-249d-46d7-95dc-db4fcff789ec.png)